internal/storage/debug_bucket.go

```diff
@@ -163,8 +163,12 @@ func (b *debugBucket) CreateChunkUploader(
 	ctx context.Context,
 	req *gcs.CreateChunkUploaderRequest,
 	writeChunkSize int,
-	progressFunc func(int64)) (gcs.ChunkUploader, error) {
-	return nil, fmt.Errorf("not implemented yet")
+	progressFunc func(int64)) (uploader gcs.ChunkUploader, err error) {
+	id, desc, start := b.startRequest("CreateChunkUploader(%q)", req.Name)
```

Comment:
please check if we should do similar to NewReader.

Reply:
Good point. Added debugChunkUploader similar to debugReader.